float64 throttle
float64 steering
float64 brake
# string vehicle_name